spec/support/config/development.yml

```diff
@@ -1,4 +1,18 @@
-colorize_logging: true
+logging:
+  severity: debug
```

Comment:
Should we perhaps put a comment here about other severity levels?

Reply:
There is information about the severity levels in **config/application.cr**